lib/gui/app.js

```diff
@@ -154,15 +154,15 @@ app.run((AnalyticsService, OSWindowProgressService) => {
   });
 });
 
-app.run(($timeout, DriveScannerService, DrivesModel, ErrorService) => {
+app.run(($timeout, DriveScannerService, ErrorService) => {
   DriveScannerService.on('drives', (drives) => {
 
     // Safely trigger a digest cycle.
     // In some cases, AngularJS doesn't acknowledge that the
     // available drives list has changed, and incorrectly
     // keeps asking the user to "Connect a drive".
     $timeout(() => {
-      DrivesModel.setDrives(drives);
+      availableDrives.setDrives(drives);
```

Comment:
I'm afraid I'm a bit confused. Here we're explicitly setting a list of drives...